### PR TITLE
Fieldset and Legend Accessibility issue fix

### PR DESCRIPTION
**Pull request by santu412**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-7781

### Change description

DAC - Fieldset and Legend / Features/Roc-7781

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/views/macro/date.njk [hunk 1]
```diff
@@ -11,19 +11,33 @@
...
       <div class="form-group form-group-day">
+        <legend> 
+          <span class="visuallyhidden">
+          Enter Day of your date of birth</span>
+      </legend>
         <label class="form-label" id="{{ name }}[day][label]" for="{{ name }}[day]">{{ t('Day') }}</label>
         <input class="form-control {% if dayError or dateError %} form-control-error {% endif %}" id="{{ name }}[day]" name="{{ name }}[day]" value="{{ form.valueFor(name).day | default('') }}" type="number" aria-describedby="dob-hint">
       </div>
       <div class="form-group form-group-month">
+        <legend>
```
Comment (prudhvi-v) [inline src/main/views/macro/date.njk:25]: Ideally only one legend is allowed i believe. This needs to be addressed

File: src/main/views/macro/table.njk [hunk 2]
```diff
@@ -20,7 +20,12 @@
     </div>
     {% if changeLink %}
       <div class="column-one-quarter change-link-column">
-        <a href="{{ changeLink | safe }}" class="bold">{{ t('Change') }} <span class="visuallyhidden">{{ label | lower }}</span></a>
+        {% if (label === 'Full name') %}
```
Comment (prudhvi-v) [inline src/main/views/macro/table.njk:23]: There are several such scenarios in both claimant and defendant journey. These needs to be addressed

File: src/main/views/macro/date.njk [hunk 3]
```diff
@@ -11,6 +11,11 @@
     {% if monthError %}<span class="error-message">{{ t(monthError) }}</span>{% endif %}
     {% if yearError %}<span class="error-message">{{ t(yearError) }}</span>{% endif %}
     {% if dateError %}<span class="error-message">{{ t(dateError) }}</span>{% endif %}
+    <fieldset id="dateofbirth[label]">
```
Comment (pradhap87) [inline src/main/views/macro/date.njk]: This will cause ally test case failure, So please can you review this changes before merging this PR

Comment (pradhap87) [reply src/main/views/macro/date.njk]: I have removed the fieldset to make the ally test case to pass. Please review my changes before proceeding further.

Comment (pradhap87) [reply src/main/views/macro/date.njk]: I have removed the fieldset to make the ally test case to pass. Please review my changes before proceeding further.

File: src/main/views/macro/date.njk (hunk 1 above)
Comment (pradhap87) [reply src/main/views/macro/date.njk]: Yes, actually we no need to keep **visuallyhidden** inside the legend, if this not intentional it's good to remove them.

**Comment by pradhap87** [comment]
Closing this PR due to issue in the AKS deployment. Will open one new PR.

**Pull request closed without merging**